src/store/ChartStore.js

```diff
@@ -247,7 +248,15 @@ class ChartStore {
                 } else {
                     this.changeSymbol(this.defaultSymbol);
                 }
+
                 this.setLayoutData(context);
+                /**
+                 * Updating market close status each 10 minute
+                 */
+                this.updateMarketClosedStatus(api);
+                setInterval(() => {
+                    this.updateMarketClosedStatus(api);
+                }, 10 * 60 * 1000);
```

Comment:
I think why QA is not seeing the update is because of the update interval. Not entirely unexpected, since we are updating the trade times every 10 minutes. What if QA sets the trade close time as 10:32 or 11:57?

@aminroosta I've brought this up before. Either we update every minute, or use a timer.